src/mito2/src/cache.rs

```diff
@@ -110,6 +106,19 @@ impl CacheManager {
         None
     }
 
+    /// Gets cached [ParquetMetaData] from memory cache.
+    pub fn get_parquet_meta_data_mem(
```

Comment:
When to use `get_parquet_meta_data_mem` and when to use `get_parquet_meta_data` 🙊 

(BTW, why not "metadata"

Reply:
> BTW, why not "metadata"

The struct name is `MetaData`.